src/main/webapp/user_script.js

```diff
@@ -15,6 +15,8 @@ function loadProfile() {
         const bioSection = document.getElementById('bio');
         bioSection.innerHTML = profile.bio;
         bioSection.value = profile.bio;
+        const profileImage = document.getElementById("avatar");
+        profileImage.src = "https://icotar.com/initials/" + profile.username;
```

Comment:
Lets do what Rodrigo suggested and only include the first letter of the username in the URL.

Reply:
I will be fixing the icon issue in a separate PR.

Reply:
Sounds good